IFC4x3/Properties/s/SeismicResistance_17fvYJ58159gRnjvJMIylv/Documentation.md

```diff
@@ -0,0 +1 @@
+Maximum level of earthquake that the equipment complies with. The value indicates earthquake intensity measured in Richter scale.
```

Comment:
Mximum magnitude or scale seems better wording. Let us resolve this together in the meeting.